models/articles.models.js

```diff
@@ -15,6 +16,19 @@ function fetchArticles() {
     })
 }
 
-module.exports = {fetchArticleById, fetchArticles};
+function updateArticleById({article_id}, {inc_votes}) {
+    if(isNaN(inc_votes)) return Promise.reject({status: 400, msg: 'inc_votes must be a number'});
+    return db.query(`
+    UPDATE articles
+    SET votes = votes ${calculateVotes(inc_votes)}
```

Comment:
You could be using the `$` binding syntax here which lets PSQL do all the hard work for you on sql injection and data type security